.vscode/settings.json

```diff
@@ -1,4 +1,15 @@
 {
+  "remark.format": {
+		"plugins": [
+			"github"
+		],
+		"rules": {
+			"closeAtx": true
+		},
+		"github": {
+			"repository": "https://github.com/mrmlnc/vscode-remark"
+		}
+	},
```

Comment:
Is this the right indentation?

Reply:
The indentation only looks like that on GitHub.